libs/remote_storage/src/s3_bucket.rs

```diff
@@ -98,18 +99,20 @@ impl S3Bucket {
             .set_max_attempts(Some(1))
             .set_mode(Some(RetryMode::Adaptive));
 
-        let mut config_builder = Config::builder()
+        let mut config_builder = Builder::default()
+            .behavior_version(BehaviorVersion::v2023_11_09())
```

Comment:
Need this over at s3_scrubber/lib.rs

Reply:
[437c8ec](https://github.com/neondatabase/neon/pull/5976/commits/437c8eca5ccc030f66202afba44e38ef563e4f20)